client/src/pages/pricing/index.js

```diff
@@ -33,7 +33,12 @@ const PricingPage = () => {
 
     return (
       <div key={name}>
-        <img src={image} alt={name} />
+        <a href="https://www.freedomrains.com">
```

Comment:
Great job on this code! Just a quick note: I don't think lines 36 and 41 are necessary here. Having lines from 37 to 39 are enough for 3 solutions images to link each corresponding pricing component.